src/Shared/Proto/opentelemetry/proto/logs/v1/logs.proto

```diff
@@ -208,4 +210,18 @@ message LogRecord {
   //   - the field is not present,
   //   - the field contains an invalid value.
   bytes span_id = 10;
+
+  // A unique identifier of event category/type.
+  // All events with the same event_name are expected to conform to the same
+  // schema for both their attributes and their body.
+  //
+  // Recommended to be fully qualified and short (no longer than 256 characters).
+  //
+  // Presence of event_name on the log record identifies this record
+  // as an event.
+  //
+  // [Optional].
+  //
+  // Status: [Development]
+  string event_name = 12;
```

Comment:
@rajkumar-rangaraj, should be add support for this in the OTLP exporter?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol#environment-variables-1
It'll be bit awkward that EventName gets exported automatically, but EventId still requires opt-in. But still, I think that is the way forward.

Reply:
@Kielek Not before the next stable release. This was not a part of the earlier code. I see you had created an issue to track this item; we could add it after this stable release.